pkg/cmd/cli/describe/printer.go

```diff
@@ -138,6 +140,11 @@ func NewHumanReadablePrinter(noHeaders, withNamespace, wide bool, showAll bool,
 	return p
 }
 
+func NewHumanReadablePrinterWithResourceAliases(resourceAliases map[string]string, noHeaders, withNamespace, wide bool, showAll bool, showLabels bool, absoluteTimestamps bool, columnLabels []string) *kctl.HumanReadablePrinter {
+	resourceShortForms = resourceAliases
```

Comment:
@smarterclayton Should a PR for this change wait to be merged after the next rebase?